shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -95,14 +95,14 @@
     public static final String CACHE_GET_LATENCY = "segmentstore.cache.get_latency_ms";         // Histogram
```

Comment:
I think that this should be fixed as well, eight? Please, double check all the fields with a `Histogram` comment.

Reply:
This belongs to RocksDB which reads as "rocksdb.segmentstore.cache.get_latency_ms", there is no duplication.

Reply:
Is the goal to reduce "duplication" or to improve "consistency".    This PR certainly doesn't look very consistent.